internal/service/studies/main.go

```diff
@@ -185,6 +185,7 @@ func (s *Service) createStudy(user types.User, studyData openapi.StudyCreateRequ
 		OwnerUserID:                user.ID,
 		Title:                      studyData.Title,
 		DataControllerOrganisation: studyData.DataControllerOrganisation,
+		ApprovalStatus:             "Incomplete", // Initial status is "Incomplete" until the contract and assets are created
```

Comment:
let's use the enum value here https://github.com/ucl-arc-tre/portal/blob/59c175a0ab211b4993a4f8d980816072d198d9fd/internal/openapi/web/main.gen.go#L50